File: Beginner/Input_age.java
import java.util.*;

public class New {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int age = sc.nextInt();

        if(age > 18) {
            System.out.println("adult");
        }
        else if (age < 10) {
            System.out.println("child");
        }
        else {
            System.out.println("teen");
        }
    }
}

File: Beginner/InputAge.js
 function cal(age) {
      if (typeof age != "number") return console.log("Please Enter a number");
      else if (age > 18) return console.log("adult");
      else if (age < 10) return console.log("child");
      else return console.log("teen");
    }
cal(10);
